pom.xml

```diff
@@ -138,6 +138,7 @@
         <version.spotbugs>4.2.3</version.spotbugs>
         <version.spotbugs.maven>4.2.2</version.spotbugs.maven>
         <version.surefire>3.0.0-M5</version.surefire>
+        <project.java.target>1.8</project.java.target>
```

Comment:
I think that this is now the only location where the java version is defined in the pom.  Note that there are a couple of named references for plugin configuration that are not extracted.

Overall, this should hopefully make future updates easier to manage.